#4dpzbk / - #2bj0dv - Add meta tags (#395)
pupil-labs/pupil-docs: src/core/software/recording-format.md

---
permalink: /core/software/recording-format

> description: Each recording contains a number of files. The exact number and names depend on which Pupil Core headset is connected and which features are enabled during the recording.

Reviewer: ```suggestion
description: Pupil Core recording format. Learn about the files contained within a Pupil Core recording. 
```
